src/SymbolServer.jl

```diff
@@ -51,7 +55,7 @@ function get_general_pkgs()
                 reg = Pkg.Types.read_registry(joinpath(r.path, "Registry.toml"))
                 return reg["packages"]
             end
-            return Dict{UUID, PkgEntry}()
+            return Dict{String, Any}()
```

Comment:
Now this empty Dict is the same type as `reg["packages"]`. The difference was already handled in `remove_non_general_pkgs!`.